doc/data-mining-library/source/tutorial/code/data-feature-selection.py

```diff
@@ -1,8 +1,7 @@
 import Orange
 
 data = Orange.data.Table("iris.tab")
-new_domain = Orange.data.Domain(list(data.domain.attributes[:2]),
-                                data.domain.class_var)
+new_domain = Orange.data.Domain(list(data.domain.attributes[:2]), data.domain.class_var)
```

Comment:
This is not PEP8-compliant. I'd prefer having lines shorter than 80 chars.

Reply:
I guess Black defaults to 88 character length:
https://github.com/psf/black#line-length

Where do you recommend that I add the configuration, so that Black will respect the 80 character length limit?

Reply:
Some reasoning: https://www.reddit.com/r/Python/comments/9uxo9f/do_you_use_black_with_its_default_line_length_of/

Apparently you can set it with config override `--line-length=79`. I am not using Black though, so haven't tried it myself.

Reply:
OK, fixed.